packages/contentful--app-scripts/lib/utils.js

```diff
@@ -81,7 +100,31 @@ function getActionsManifest() {
   ----------------------------`);
     console.log('');
 
-    return manifest.actions.map((action) => ({ parameters: [], ...action })); // adding required parameters
+    const actions = manifest.actions.map((action) => {
+      const allowedNetworks = Array.isArray(action.allowedNetworks) ? action.allowedNetworks : [];
```

Comment:
Can map here instead of doing it twice below.
```suggestion
      const allowedNetworks = Array.isArray(action.allowedNetworks) ? action.allowedNetworks.map(removeProtocolFromUrl) : [];
```